cookbook/add-tabs.rst

```diff
@@ -0,0 +1,300 @@
+Adding tabs to Sulu admin interface
+===================================
+
+.. note:: It is recommended to read `Extend admin Ui`_ beforehand to get a better understandig of how Sulu admin classes work.
+
+
+This tutorial will walk you through the process of adding extra tabs to the user interface of Sulu.
+
+This could be useful in a few situations, a few examples below:
+
+- Saving extra data for pages
+- Saving extra data in webspaces
+- Saving extra data for users
+
+In this example we'll be adding a "Socials" tab to the PageAdmin of Sulu.
+
+.. figure:: ../img/socials-tab.png
+
+Create the form for this view
+-----------------------------
+You have to create the form that is rendered in this view. For this we create an XML file like your used to.
+Create your form: ``config/forms/page_socials.xml``. Notice how we use slashes in the names of the properties, this will be of later use.
```

Comment:
"Therefore create a form at ``config/forms/page_socials.xml``:"

The notice could be an actual note using the `.. note::` syntax. Also explain what it is doing (returning the values in the given hierarchy) instead of just saying that it will be of use later.